src/configuration.cc

```diff
@@ -46,6 +46,9 @@ configuration::configuration(broker_options opts) : options_(std::move(opts)) {
                  "maximum number of entries when recording published messages");
   // Override CAF default file names.
   set("logger.file-name", "broker_[PID]_[TIMESTAMP].log");
+  auto log_verbosity = caf::get_if<caf::atom_value>(this, "logger.verbosity");
+  if (!log_verbosity)
+    set("logger.verbosity", caf::atom("quiet"));
```

Comment:
I would set this unconditionally here. The only thing running before the constructor of `configuration` is the constructor of `caf::actor_system_config`. In fact, if `logger.verbosity` is not set, then we *should* set it here because CAF otherwise picks the default value (which depends on the build config).

Unconditionally setting this to `quiet` here means that users can still pass a different value via CLI or config file, which I believe is the behavior you're looking for. No explicit user override = no logs.

Reply:
Oh, one more thing: I wouldn't use the legacy `verbosity` setting any more (we're probably going to deprecate it soon). Instead:

```c++
set("logger.file-verbosity", caf::atom("quiet"));
set("logger.console-verbosity", caf::atom("quiet"));
```